roles/confluent.control_center/tasks/main.yml

```diff
@@ -14,6 +14,46 @@
     - ansible_os_family
     - ansible_fqdn
 
+- name: Get Package Facts
+  package_facts:
+  when: installation_method == "package"
+
+- name: Stop Control Center Service if Upgrading Packages
```

Comment:
I think the conditions to stop the service are:
- packages need to change versions
- ansible_facts.packages['confluent-kafka-2.12'] is defined and confluent_server_enabled == true
- ansible_facts.packages['confluent-kafka'] is defined and confluent_server_enabled == true
- ansible_facts.packages['confluent-server'] is defined and confluent_server_enabled == false

Next, If package versions need to change -> `yum remove confluent-*`

Else yum remove confluent-kafka/server appropriately

And then the regular install will take over and put things right. This would achieve downgrading and toggling confluent_server_enabled

Reply:
@domenicbove I have a few other things to fix first, but I will be looking into this a bit later today.